Sources/Jetworking/Client/Client.swift

```diff
@@ -375,27 +385,36 @@ public final class Client {
         }
     }
 
+    private func handleSuccessVoid<ResponseType>(currentURLResponse: HTTPURLResponse, endpoint: Endpoint<ResponseType>?, data: Data?, completion: @escaping RequestCompletion<ResponseType>) {
+        if ResponseType.self is Void.Type {
+            return enqueue(completion(currentURLResponse, .success(() as! ResponseType)))
+        }
+
+        enqueue(completion(currentURLResponse, .failure(APIError.unexpectedError)))
+    }
+
+    func handleSuccessDecodable<ResponseType: Decodable>(currentURLResponse: HTTPURLResponse, endpoint: Endpoint<ResponseType>?, data: Data?, completion: @escaping RequestCompletion<ResponseType>) {
```

Comment:
Wasn't it possible to extract these functions (handleSuccessVoid, handleSuccessDecodable) into separate files? For example as strategy or something else? This could reduce the code in this file since we are over 400 lines of code and would separate the functionality.

Reply:
The whole response handling is now moved to a dedicated file.